keyboards/mechlovin/infinity87/rev2/info.json

```diff
@@ -1,13 +1,27 @@
 {
     "keyboard_name": "Infinity87 Rev.2",
-    "manufacturer": "Mechlovin.Studio",
     "url": "",
-    "maintainer": "Mechlovin' Studio",
     "usb": {
-        "vid": "0x4D4C",
         "pid": "0x8702",
         "device_version": "0.0.1"
     },
+    "matrix_pins": {
+        "cols": ["NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN", "NO_PIN" ],
+        "rows": [ "D5", "D2", "D4", "D3", "D0", "D1"],
+        "custom": true,
+        "custom_lite": true
+    },
```

Comment:
For custom matrix boards, do not move the pin config to info.json.

Reply:
I moved the pin back to config.h, thanks.